packages/playground/_stories/fiori/SideNavigation/SideNavigation.stories.ts

```diff
@@ -155,10 +155,14 @@ ToolLayout.decorators = [
 </div>
 
 <script>
-	const sidenav = document.querySelector("ui5-side-navigation");
```

Comment:
switch `const` with `var` and that will fix the issue with redeclaring the variable